frontend/smart-kickers-game/src/components/Game/GameStatistics/GameStatistics.js

```diff
@@ -2,8 +2,21 @@ import React from 'react';
 import './GameStatistics.css';
 import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
 import { Button } from '../../Button/Button.js';
+import { getStatistics } from '../../../apis/getStatistics.js';
+import { useEffect, useState } from 'react';
+import { TeamID } from '../../../constants/score.js';
 
 function GameStatistics({ finalScores, onNewGameRequested }) {
+  const [data, setData] = useState({});
+  useEffect(() => {
+    getStatistics().then((result) => {
+      if (result?.error) alert(result.error);
+      setData(result.data);
```

Comment:
```suggestion
      setData(result);
```
If you accept changes in `getStatistics` you should also change this line in that way.

Reply:
Done